### PR TITLE
car wash station methods

### PR DESCRIPTION
**Pull request by A-n-a-s-t-a-s-i-i-a**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,43 @@
...
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars_list: list) -> float:
```
Comment (Oleksl888) [inline app/main.py:22]: Also mention data type of the list contents

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,43 @@
...
+        return income
+
+    def calculate_washing_price(self, car_brand: Car) -> float:
+        return round(car_brand.comfort_class
+                     * (self.clean_power - car_brand.clean_mark)
+                     * self.average_rating
+                     / self.distance_from_city_center, 1)
+
+    def wash_single_car(self, car_brand: Car) -> None:
+        car_brand.clean_mark = self.clean_power
```
Comment (Oleksl888) [inline app/main.py:37]: This method missing condition as per task definition:

> wash_single_car - method, that washes a single car, so it should have clean_mark equals wash station's clean_power, if wash_station.clean_power is greater than car.clean_mark;

**Review by Oleksl888 — CHANGES_REQUESTED** [review]

**Review by Oleksl888 — CHANGES_REQUESTED** [review]
please change

**Review by Oleksl888 — CHANGES_REQUESTED** [review]
request to sync on platform

**Review by oliasymak24 — COMMENTED** [review]
You closed your pull request

File: app/main.py
Comment (LiudmylaKulinchenko) [inline app/main.py:1]: closed PR

**Pull request closed without merging**